### ISSUE TITLE
Upload cannot be cancelled

### ISSUE THREAD

**Issue by hihain** [UnigramDev/Unigram#1961]
Once a file gets uploaded, clicking the x button does not have any effect.

![ezgif-7-97901d073c9a](https://user-images.githubusercontent.com/1473059/90449696-3f76b480-e0e8-11ea-8c00-93f93c3bc5ef.gif)

Further error: After successful upload, the image does not get swapped for the play icon. Only after an UI update like leaving the chat or uploading the same file again, the second upload shows the correct icon (the first stays wrong until reopening the chat). Should I open another issue for this?
